src/Native/System.Security.Cryptography.Native/pal_cms.h

```diff
@@ -7,14 +7,25 @@
 #include <openssl/cms.h>
 
 /*
+These values should be kept in sync with System.Security.Cryptography.Pkcs.SubjectIdentifierType.cs
+*/
+enum SubjectIdentifierType : int32_t
+{
+    Unknown = 0,                // Use any of the following as appropriate
+    IssuerAndSerialNumber = 1,  // X509IssuerSerial
+    SubjectKeyIdentifier = 2,   // SKI hex string
+    NoSignature = 3             // NoSignature
+};
+
+/*
 Shims the d2i_CMS_ContentInfo method
 */
 extern "C" CMS_ContentInfo* CryptoNative_CmsDecode(const uint8_t* buf, int32_t len);
 
 /*
-Shims the CMS_decrypt method
+Shims the CMS_decrypt method and adds the implicit skid to the cert if necessary
```

Comment:
Rather than "add", which suggests that it might modify `cert` you may want "provides an implicit SKID for the cert if necessary".